packages/babel-parser/src/plugin-utils.js

```diff
@@ -87,6 +87,11 @@ export function validatePlugins(plugins: PluginList) {
   }
 
   if (hasPlugin(plugins, "moduleAttributes")) {
+    if (process.env.BABEL_8_BREAKING) {
+      throw new Error(
+        "`moduleAttributes` has been removed in Babel 8, please use `importAssertions` parser plugin, or `@babel/plugin-syntax-import-assertions`.",
+      );
+    }
     if (hasPlugin(plugins, "importAssertions")) {
```

Comment:
```suggestion
    else if (hasPlugin(plugins, "importAssertions")) {
```

so that the plugin we use to remove/evaluate `BABEL_8_BREAKING` can more remove this code (since it will be in the `else` branch).